dev-libs/openssl/openssl-1.0.2f.recipe

```diff
@@ -15,13 +15,13 @@ HOMEPAGE="https://www.openssl.org/"
 COPYRIGHT="1995-1998 Eric Young
 	1998-2016 The OpenSSL Project."
 LICENSE="OpenSSL"
-REVISION="1"
-SOURCE_URI="https://www.openssl.org/source/openssl-${portVersion}.tar.gz"
+REVISION="2"
+SOURCE_URI="https://www.openssl.org/source/openssl-$portVersion.tar.gz"
 CHECKSUM_SHA256="932b4ee4def2b434f85435d9e3e19ca8ba99ce9a065a61524b429a9d5e9b2e9c"
-PATCHES="openssl-$portVersion.patch"
+PATCHES="openssl-$portVersion.patchset"
 
 ARCHITECTURES="x86_gcc2 x86 x86_64 ?arm ?ppc"
-SECONDARY_ARCHITECTURES="x86_gcc2 x86 ?arm ?ppc"
+SECONDARY_ARCHITECTURES="x86_gcc2 x86 ?x86_64 ?arm ?ppc"
```

Comment:
Also fixed. BTW, are `x86_gcc2` and `x86` the only archivetures allowed in SECONDARY_ARCHITECTURES?
If the answer is yes then I will fix (in a separate PR) all other recipes I have been editing recently.